README.md

```diff
@@ -127,38 +129,62 @@ heartbeats:
 
 ```
 
-You can also create and update monitors by calling `Monitor.put`.
+### Using `Cronitor::Monitor.put`
+
+You can also create and update monitors by calling `Cronitor::Monitor.put`. This method can handle multiple monitors at once and supports various configurations and options.
+
 
+#### Usage
 ```ruby
 require 'cronitor'
 
+# Define monitors as an array of hashes
 monitors = [
   {
     type: 'job',
     key: 'send-customer-invoices',
     schedule: '0 0 * * *',
-    assertions: [
-        'metric.duration < 5 min'
-    ],
+    assertions: ['metric.duration < 5 min'],
     notify: ['devops-alerts-slack']
   },
   {
     type: 'check',
     key: 'Cronitor Homepage',
-    request: {
-        url: 'https://cronitor.io'
-    },
+    request: { url: 'https://cronitor.io' },
     schedule: 'every 60 seconds',
-    assertions: [
-        'response.code = 200',
-        'response.time < 600ms',
-    ]
+    assertions: ['response.code = 200', 'response.time < 600ms']
```

Comment:
Would you mind reverting these changes? Laying it out like this is done for readability purposes.

Reply:
updated to separate lines